app/src/main/java/com/arielwang/workoutlogger/features/component/ScreenButton.kt

```diff
@@ -0,0 +1,37 @@
+package com.arielwang.workoutlogger.features.component
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun ScreenButton(
+    buttonConstraintLayout: Modifier,
+    @StringRes buttonText: Int,
```

Comment:
Instead of passing in a string id, it would be better to just pass in a string. It doesn't matter on single-module projects because you only have one strings.xml, but in multi-module projects, you will have many strings.xml and R files. If this component is shared between modules, you can't find the correct string in the wrong R file